packages/widgets/src/lib/components/Popover/Popover.svelte

```diff
@@ -1,7 +1,12 @@
+<!-- 
+	In https://github.com/huggingface/huggingface.js/pull/567, this component was changed significantly from the original one in moon/Popover.svelte.
```

Comment:
btw, I tried using the in moon/Popover.svelte. However, wasn't able to make it work out of the box (i.e. there were issues on alignments & the popup was looking in unexpected behaviours) without changing the Popover.svelte codebase

Reply:
Indeed, I quickly tried and there's a difference on how the anchor  and opening is handled. Here the anchor is outside the popover. 
I suggest to merge this PR as it is and refactor it when integrating back to moon